src/net_processing.cpp

```diff
@@ -80,7 +82,19 @@ static_assert(INBOUND_PEER_TX_DELAY >= MAX_GETDATA_RANDOM_DELAY,
 "To preserve security, MAX_GETDATA_RANDOM_DELAY should not exceed INBOUND_PEER_DELAY");
 /** Limit to avoid sending big packets. Not used in processing incoming GETDATA for compatibility */
 static const unsigned int MAX_GETDATA_SZ = 1000;
-
+// Used to convert reconciliation q to an int
+static constexpr uint64_t Q_PRECISION = 2 << 12;
```

Comment:
```suggestion
static constexpr double Q_PRECISION{2 << 12};
```

This is used to map an int to a small floating point. So it would probably be safer and avoid accidental integer truncation if this was already a double at compile time